Makefile

```diff
@@ -117,11 +123,19 @@ ifneq ($(SKIP_TESTS), true)
 endif
 
 .PHONY: test-scorecard
-test-scorecard: destroy_cryostat_cr undeploy uninstall
+test-scorecard: clean-scorecard
```

Comment:
I guess since the scorecard tests run in a separate namespace now, it's not necessary to clear out any previous installation? Would it make sense to fail early on this run if the namespace already exists, or even exists and is not empty?

Reply:
The namespace could be non-empty if any of the test steps encounter an error. In this case, the recipe would exit before running the `delete namespace` command. I suppose it's probably safer to make the user delete the namespace manually.

Reply:
I am thinking whether instead of deleting the namespace before tests, we could clean up only operator resources with `operator-sdk cleanup` and ignore if namespace does not exist?

Reply:
As we get more complicated tests, there could be additional objects that get created that aren't cleaned up by `operator-sdk cleanup`. In order to ensure the tests have a clean slate, it's ideal to wipe out the namespace. A fresh cluster is even better, but not the most practical.

Reply:
Ahh right! Thanks for explaining :D

Reply:
Added a check for the namespace which prompts the user to delete it, if it exists.